.github/workflows/cicd.yaml

```diff
@@ -62,18 +63,17 @@ jobs:
       - name: cleanup
         if: always()
         run: |
-          source /etc/profile.d/enable-rh.sh
           git clean -dfx
 
   sql_sdk_test:
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/4paradigm/hybridsql:0.3.0
+      image: ghcr.io/4paradigm/hybridsql:0.4.0
```

Comment:
well, we have conflicts #266

Reply:
Yep. We will sync the update after merging #266